tests/e2e/network/suite.go

```diff
@@ -0,0 +1,46 @@
+package network
+
+import (
+	"time"
+
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const subnetName = "e2eSubnetTest"
+
+var _ = ginkgo.Describe("[Network]", func() {
+	ginkgo.AfterEach(func() {
+		commands.CleanNetwork()
+		err := utils.DeleteConfigs(subnetName)
+		gomega.Expect(err).Should(gomega.BeNil())
+	})
+
+	ginkgo.It("can stop and restart a deployed subnet", func() {
+		genesis := "tests/e2e/genesis/test_genesis.json"
+
+		commands.CreateSubnetConfig(subnetName, genesis)
+		deployOutput := commands.DeploySubnetLocally(subnetName)
+		_, err := utils.ParseRPCFromDeployOutput(deployOutput)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		commands.StopNetwork()
+		restartOutput := commands.StartNetwork()
+		rpc, err := utils.ParseRPCFromRestartOutput(restartOutput)
+		gomega.Expect(err).Should(gomega.BeNil())
+
+		err = utils.SetHardhatRPC(rpc)
```

Comment:
maybe you can check some snapshot preservation, like rpc equalitty before/after

Reply:
added a more sophisticated check